### ISSUE TITLE
No longer functioning?  Can you suggest steps to troubleshoot?

### ISSUE THREAD

**Issue by tarah34** [magbicaleman/open-in-browser#39]
This is a fantastic package, one that I rely on all the time.  However starting on 3/27/17 for some reason, I can no longer get open-in-browser to work in dropdown menu, either in file, or, in the file tree.  I uninstalled and reinstalled the package.  I didn't have nuclide and noticed in the 0.4.7 version you mentioned one should have it installed for open-in-browser to work. I installed nuclide and then installed open-in-browser again, but still it doesn't work.  Any suggestions?  Thanks!

**Comment by magbicaleman** [comment]
Hey @tarah34, thanks for opening up this issue. I haven't actively been keeping track with my plugin which is a shame. I'm not sure what could be the cause of the problem. 

I wonder if you disabled all of other plugins, if you have any; and just tried the open-in-browser one, it's possible another plugin might be interfering with it.

You don't need nuclide, I may have reported that incorrectly. If I'm remembering that correctly, the issue was that it wasn't opening in nuclide and a pull-request solved that problem.

There is also a Pull request that's been open for close to a year, that I reviewed but never added on to the plugin, which may fix a lot of other problems.

**Comment by tarah34** [comment]
Hi @magbicaleman, I solved the issue and your plugin remains a daily can't-live-without tool.  The problem was that I had changed the default "open with" on .html files to Atom instead of a web browser and that made the plugin stop functioning.  Once I changed that back, your plugin in Atom worked again.  Sorry to have raised a flag, but by communicating the problem, I managed to create the energy to solve the problem :)
